packages/mobile/src/services/query-client.ts

```diff
@@ -10,7 +10,8 @@ export const queryClient = new QueryClient({
       refetchOnWindowFocus: true,
       refetchOnReconnect: true,
       refetchOnMount: true,
-      throwOnError: env.ENVIRONMENT === 'development' // feature-tan-query TODO: remove before going to main?
+      throwOnError: env.ENVIRONMENT === 'development', // feature-tan-query TODO: remove before going to main?,
+      experimental_prefetchInRender: true
```

Comment:
Curious why we're enabling this?

Reply:
afaict the timing of either the cache priming or chained hook usage we're doing falls under the category of prefetchInRender. Without this, we get console errors from tan-query saying the feature isn't on but we're using it like it is.

ofc I'm not 100% sure if it's safe. Need to keep an eye on it, but in my testing the only lineup that uses this (listening history) was working normally

Reply:
oh weird yeah lets keep an eye on this